packages/ui/src/components/auth/SignForm.module.css

```diff
@@ -61,7 +61,9 @@
 .form-width {
   width: 245px;
 }
-
+.resendLink {
+  font-size: 15px;
```

Comment:
Please use a standard font value. Check other text and their font sizes, don't put arbitrary values here